src/collective/fingerpointing/upgrades/configure.zcml

```diff
@@ -0,0 +1,29 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:gs="http://namespaces.zope.org/genericsetup">
+
+  <include file="profiles.zcml" />
+
+  <gs:registerProfile
```

Comment:
well, you have a point indeed but; I think this package is not going to be so complex in the near future and that's why we always test upgrade steps.

if you prefer to do it this way, don't forget to hide the upgrade step profile so is not listed neither in QuickInstaller, nor in portal_setup.